### ISSUE TITLE
Make the pkg file executable

### ISSUE THREAD

**Issue by louies0623** [helloSystem/hello#194]
Allow users to install software using pkg files, and no complicated terminal is required, just a few clicks with the mouse can be installed.

**Comment by kettle-7** [comment]
The idea is that you don't *need* packages, or installation, you just download the app and there it goes.

**Comment by probonopd** [comment]
True. On helloSystem, we prefer [Application Bundles](https://hellosystem.github.io/docs/developer/application-bundles.html).

Nevertheless, we could do something similar to 

![image](https://user-images.githubusercontent.com/2480569/125225699-0337de00-e2d0-11eb-85a1-b992bb3f7693.png)

when one double-clicks a txz file. 

We could probably implement it in https://github.com/helloSystem/ISO/blob/experimental/overlays/uzip/hello/files/usr/local/libexec/app-placeholder.

When a txz file is the first argument, then we could show

> [Package icon]
> The software inside this package needs to be installed before it can be used. 
> 
> Do you want to install it now?

It's just Python, any volunteers? ;-)
